app-vim/alternate/alternate-2.18-r2.ebuild

```diff
@@ -1,15 +1,16 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
 
 inherit vim-plugin
 
 DESCRIPTION="vim plugin: quickly switch between .c and .h files"
-HOMEPAGE="https://github.com/vim-scripts/a.vim"
+HOMEPAGE="https://github.com/vim-scripts/a.vim
+https://www.vim.org/scripts/script.php?script_id=31"
 SRC_URI="
-	https://github.com/vim-scripts/a.vim/archive/${PV}.tar.bz2 -> ${P}.tar.bz2
-	http://www.vim.org/scripts/download_script.php?src_id=6347 -> ${PN}.txt
+	https://github.com/vim-scripts/a.vim/archive/${PV}.tar.bz2 -> ${P}.tar.gz
```

Comment:
Wait. Why is the extension changing?

Reply:
Sorry, i forgot to change first bz2 too.
The reason i've changed it to bz2 was simply because the tar.bz2 package doesn't exist, while the tar.gz does.
I've fixed my mistake and rebase this PR. Please have a look.